graviton/inspector.py

```diff
@@ -384,7 +385,7 @@ def from_single_response(
         dryrun_resp: dict,
         args: Sequence[PyTypes],
         encoded_args: List[ArgType],
-        abi_type: Optional[abi.ABIType] = None,
+        abi_type: Optional[EncodingType] = None,
```

Comment:
I wonder if it is necessary to make it `abi_type` optional, for `EncodingType` itself is a union of `Union[abi.ABIType, str, None]`, which includes `None`.

Reply:
I'll try that!

Reply:
addressed here: a3163886762012d078d8f36a222b6277a03723b2

Reply:
lgtm